src/update/NoUpdatesAvailableDialog.cpp

```diff
@@ -0,0 +1,95 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  NoUpdatesAvailableDialog.cpp
+
+  Anton Gerasimov
+
+**********************************************************************/
+
+#include "NoUpdatesAvailableDialog.h"
+
+#include "widgets/HelpSystem.h"
+#include "ShuttleGui.h"
+
+#include "prefs/PrefsDialog.h"
+#include "ui/AccessibleLinksFormatter.h"
+
+#include <wx/stattext.h>
+
+static const auto title =
+/* i18n-hint: Title of the dialog no updates available. */
+XO("No updates available");
+
+BEGIN_EVENT_TABLE(NoUpdatesAvailableDialog, wxDialogWrapper)
+    EVT_BUTTON(wxID_OK, NoUpdatesAvailableDialog::OnOk)
+END_EVENT_TABLE()
+
+NoUpdatesAvailableDialog::NoUpdatesAvailableDialog(wxWindow* parent)
+    : wxDialogWrapper(
+         parent, -1, XO("Check for Updates"), wxDefaultPosition, wxDefaultSize, wxCAPTION | wxCLOSE_BOX)
+{
+   ShuttleGui S(this, eIsCreating);
+
+   S.SetBorder(8);
+
+   S.StartVerticalLay();
+   {
+      S.AddSpace(0, 12);
+
+      S.StartHorizontalLay();
+      {
+         S.AddSpace(12, 0);
+
+         S.StartVerticalLay();
+         {
+            wxStaticText* titleCtrl = S.AddVariableText(title, false, 0, 500);
+
+            wxFont font = titleCtrl->GetFont().MakeLarger().MakeBold();
+
+            titleCtrl->SetFont(font);
+
+            AccessibleLinksFormatter preferencesMessage(
+               /* i18n-hint: %s is replaced with 'Preferences > Application'. */
+               XO("You are on the latest version of Audacity. Consider turning %s on in Preferences, if you want to be informed automatically about new versions."));
+
+            preferencesMessage.FormatLink(
+               /* i18n-hint: Hyperlink title that opens Preferences dialog on Directories page. */
```

Comment:
i18n-hint should mention that this is the object of "Consider turning on..."

So that the translator for a language like Russian, with cases, will know which grammatical ending to use.